### PR TITLE
Version on the WP overview is now link

### PR DESCRIPTION
**Pull request by mtakac2**
https://www.openproject.org/work_packages/14066

Although not ideal solution, it should be sufficient for now. It could be definitely done more abstract, but I didn't want to spend more time on this one as it will be used just permanently.

### CODE REVIEW

File: app/assets/javascripts/angular/work_packages/controllers/details-tab-overview-controller.js [hunk 1]
```diff
@@ -58,13 +64,22 @@ angular.module('openproject.workPackages.controllers')
...
-      return $scope.workPackage.embedded[property];
-    } else {
-      return getFormattedPropertyValue(property);
+    function getPropertyValue(property, format) {
+        switch(format) {
+            case VERSION_TYPE:
+                if ($scope.workPackage.props.versionId == undefined) {
+                    return;
+                }
+                var href = '/versions/' + $scope.workPackage.props.versionId;
```
Comment (myabc) [inline app/assets/javascripts/angular/work_packages/controllers/details-tab-overview-controller.js]: @mtakac code constructing the link should probably be moved to `PathHelper`. Paths shouldn't be hard -coded here.

**Comment by myabc** [comment]
To me this errs on the side of hacky. To be fair though, the existing code wasn't great either (why do we have `getFormattedPropertyValue` and `getPropertyValue`?). Tests would also be helpful here.

**Comment by mtakac2** [comment]
@myabc I agree - it isn't the best code ever. But as I've sad before: This is something that will not be part of the client in the final version and I don't think we should spent too much time on it now.

**Comment by mtakac2** [comment]
@myabc What should I test here exactly? I didn't add anything new. I have just modified a method. Controller tests are there, there are still passing - I am not sure what test should I add.

**Pull request merged**